tt_metal/hw/inc/blackhole/core_config.h

```diff
@@ -22,3 +22,23 @@ enum class AddressableCoreType : uint8_t {
     UNKNOWN   = 5,
     COUNT     = 6,
 };
+
+enum class TensixProcessorTypes : uint8_t {
+    BRISC  = 0,
+    NCRISC = 1,
+    TRISC0 = 2,
+    TRISC1 = 3,
+    TRISC2 = 4,
+
+    DM0    = 0,
+    DM1    = 1,
+    MATH0  = 2,
+    MATH1  = 3,
+    MATH2  = 4,
+};
```

Comment:
These enums dont seem to be used anywhere. Out of curiosity, why do we have two labels for the same processor types?

Reply:
I'm setting this up for the HAL, will be used there.  brisc/etc is used throughout our code today but doesn't match metal's use, for transition I think we need both, though...hmm, if we hide behind the HAL it may not matter